### PR TITLE
Use libgcrypt include dir

### PR DESCRIPTION
**Pull request by tuxmaster5000**
The include directory of libgcrypt was not used.

### CODE REVIEW

File: nasl/CMakeLists.txt [hunk 1]
```diff
@@ -121,6 +121,7 @@ else (NOT GCRYPT)
   if (GCRYPT_VERSION VERSION_LESS "1.6")
     message (SEND_ERROR "libgcrypt 1.6 or greater is required")
   endif (GCRYPT_VERSION VERSION_LESS "1.6")
+string(REPLACE "-I" "" GCRYPT_INCLUDE_DIRS "${GCRYPT_CFLAGS}")
```
Comment (jjnicola) [inline nasl/CMakeLists.txt]: Fix indentation.
```suggestion
  string(REPLACE "-I" "" GCRYPT_INCLUDE_DIRS "${GCRYPT_CFLAGS}")
```

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -55,6 +55,7 @@ else (NOT GCRYPT)
   if (GCRYPT_VERSION VERSION_LESS "1.6")
     message (SEND_ERROR "libgcrypt 1.6 or greater is required")
   endif (GCRYPT_VERSION VERSION_LESS "1.6")
+string(REPLACE "-I" "" GCRYPT_INCLUDE_DIRS "${GCRYPT_CFLAGS}")
```
Comment (jjnicola) [inline src/CMakeLists.txt]: Same here. 
```suggestion
  string(REPLACE "-I" "" GCRYPT_INCLUDE_DIRS "${GCRYPT_CFLAGS}")
```

**Comment by tuxmaster5000** [comment]
I have changed the code style as requested.

**Comment by jjnicola** [comment]
Thanks @tuxmaster5000!

**Pull request merged**